### ISSUE TITLE
Feature request: Allow use full namespaces instead of usings in generated files

### ISSUE THREAD

**Issue by krzysiek-b** [ErikEJ/EFCorePowerTools#1227]
### Currently

Seems when using some references causes generated files to be confused about attributes like Index.
<img width="467" alt="image" src="https://user-images.githubusercontent.com/6192236/146651053-31053318-df65-42e0-a7e8-9b1c2d1e035d.png">
<img width="295" alt="image" src="https://user-images.githubusercontent.com/6192236/146651332-54fc26ce-6235-42f6-8df6-abcbc57cba41.png">

Currently generated code does not compile in my scenario and I need to manually go into every file and set the namespaces to `[Microsoft.EntityFrameworkCore.Index`

### Expected

Those namespaces can be added by default to not get confused with other similar types which might be added in the using which are added.
I think that is the default way of how WinForms desiggner files are generated.

**Comment by ErikEJ** [comment]
Where does the other Index attribute come from?

Pls share a repro project.

**Comment by krzysiek-b** [comment]
<img width="295" alt="image" src="https://user-images.githubusercontent.com/6192236/146651384-6b0f5c0a-7c33-4913-8bc7-eb40e7124212.png">

**Comment by ErikEJ** [comment]
I need a repro please

**Comment by ErikEJ** [comment]
Related issue: https://github.com/dotnet/efcore/issues/23532

**Comment by krzysiek-b** [comment]
I think I have the same issue which was described in https://github.com/dotnet/efcore/issues/23532. I started using a nuget package which does use EF 6. I cannot remove or update that nuget (remove EF6 reference). 

Using full namespaces seems to be a simple solution for that problem but I do not see any willing help us in https://github.com/dotnet/efcore/issues/23532
I also noticed a workaound with am custom scaffolder https://github.com/dotnet/efcore/issues/23532#issuecomment-894608102 so I will try with that untol this gets more attention ang gets resolved.
